contrib/forms/assessment_intake/view.php

```diff
@@ -376,4 +394,4 @@
 </form>
 <?php
 formFooter();
-?>
+?> 
```

Comment:
Pure PHP should have no closing tag and an empty line. Mixed files should have an empty line, but adding a space seems odd